firewall/filesystem_storage_firewall.go

```diff
@@ -0,0 +1,41 @@
+package firewall
+
+import (
+	"errors"
+)
+
+type FilesystemAcraFirewall struct {
+
+	handlers []QueryHandler
+
+	storageDirectory string
```

Comment:
this field doesn't used anywhere in this module

Reply:
Agree.